language/lang_ext_overview.tex

```diff
@@ -0,0 +1,7 @@
+\openshmem provides ISO \Cstd bindings. \Cstd bindings are the primary language
```

Comment:
The first and second sentences sound redundant. I would drop one of the sentences, or change/reword/reorder them if you were trying to impart specific information.